### ISSUE TITLE
How to change Hyperparameters on runtime?

### ISSUE THREAD

**Issue by Antonio-Nappi** [ljvmiranda921/pyswarms#427]
* pywarm = 1.1.0
* Python version: 3.7.8
* Operating System: Windows 10

### Description
Hi everybody, I'm new in pyswarm and I'm still studying its architecture but maybe you could help me to find the right answers. I would like to implement a PSO with TVAC (Time-Varying Acceleration Coefficients) and these coefficents depends on the current iteration and the max_iteration number. Looking in the file I've seen that the velocity and position update are made in operators script. So my question is: should I define a new method in that script (maybe a "me_velocity_update") with the right coefficients and then changing the behaviour of the topology classes in order to let them use my new method? In this way how could I pass the current iteration number? Or (maybe) there is a simpler way than this which I've not considered?

**Comment by ljvmiranda921** [comment]
Hi @Antonio-Nappi , can you check the merged Pull Request above if this answers your request? 
Right now, I don't see any possible way to update coefficients during runtime except for the newly-implemented handlers. Perhaps the best way to do PSO with TVAC is to implement your own handler (following the PR above), and pass it during the init stage

**Comment by Antonio-Nappi** [comment]
Hi @ljvmiranda921, I think that with linear decay I could fix my problem. I've seen that it's an option handler and now I'm just looking at the doc to understand how to use it, but I don't find and example. Could you provide it to me? Could you also say me if it's possible to use an option handler with the GlobalBestPSO or should I make subclasses from yours?

**Comment by ljvmiranda921** [comment]
Hi @Antonio-Nappi , can you try something like this?

```python
import pyswarms as ps
from pyswarms.utils.functions import single_obj as fx
options = {'c1': 0.5, 'c2': 0.3, 'w':0.9}
optimizer = ps.single.GlobalBestPSO(
    n_particles=10, 
    dimensions=2, 
    oh_strategy="lin_decay",   # Set it from here
   options=options
)
best_cost, best_pos = optimizer.optimize(fx.sphere, iters=100)
```

**Comment by Antonio-Nappi** [comment]
Yes, I have the latest version (downloaded specifying the version inside pycharm) but I got the following error 
`TypeError: __init__() got an unexpected keyword argument 'oh_strategy'`
and can you also say me how it's possible to set boundaries for coeffients?

**Comment by ljvmiranda921** [comment]
I'm not sure if it's possible to set boundaries for coefficients. I don't think that has been implemented yet. Maybe for now you can always check the value of that coefficient, then clamp it once it's in the lower-bound?

> Yes, I have the latest version (downloaded specifying the version inside pycharm) but I got the following error
TypeError: __init__() got an unexpected keyword argument 'oh_strategy'

Oh I'm really sorry, I thought this feature has already been released in 1.2.0! For now I suggest either:
1. Doing the bleeding-edge installation: https://github.com/ljvmiranda921/pyswarms#installation
2. Wait out for next release (hopefully by the 15th of December)

**Comment by Antonio-Nappi** [comment]
Ok, thanks for your help :)
